### PR TITLE
stylo: Handle text-emphasis-position pres attrs

### PR DESCRIPTION
**Pull request by Manishearth**
Somehow slipped through the cracks.

try: https://treeherder.mozilla.org/#/jobs?repo=try&revision=e1d349e847b528be3d61ad2ebf374a791800cfbb

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16201)

### CODE REVIEW

File: components/style/properties/longhand/inherited_text.mako.rs [hunk 1]
```diff
@@ -1076,6 +1076,28 @@ ${helpers.single_keyword("text-align-last",
...
+                let vert = if kw & structs::NS_STYLE_TEXT_EMPHASIS_POSITION_RIGHT != 0 {
+                    VerticalWritingModeValue::Right
+                } else {
+                    assert!(kw & structs::NS_STYLE_TEXT_EMPHASIS_POSITION_LEFT != 0);
+                    VerticalWritingModeValue::Left
+                };
+                let horiz = if kw & structs::NS_STYLE_TEXT_EMPHASIS_POSITION_OVER != 0 {
+                    HorizontalWritingModeValue::Over
+                } else {
+                    assert!(kw & structs::NS_STYLE_TEXT_EMPHASIS_POSITION_UNDER != 0);
```
Comment (emilio) [inline components/style/properties/longhand/inherited_text.mako.rs]: nit: debug_assert

**Comment by Manishearth** [comment]
addressed

**Pull request merged**